### ISSUE TITLE
Possibility for defaults and not required in the config file

### ISSUE THREAD

**Issue by vongohren** [SpectralOps/teller#29]
# Default value for opts

I use this alot

```
  opts:
  #region: env:AWS_REGION    # you can get env variables with the 'env:' prefix
  prefix: env:PREFIX
```

But it would be great with a way to say that something was default?

# dotenv not required
I also use this alot
```
dotenv:
    env_sync:
      path: ./.env.local
```
It would be great to have an option to mark this as not required meaning that it does not crash when there is not file found. Because this particular example is only used for local override of config

**Comment by jondot** [comment]
Hi @vongohren, let's see if I understand it right:

1. Default value for opts. This means in case I'm trying to pull variables from `env` and they don't exist -- use a default value that a user gives?
2. dotenv not required. Got it so in case a file does not exist, the fetched values are simply empty/nonexisting (we could issue a warning just in case)

Does this make sense?

**Comment by vongohren** [comment]
1. Very correct
2. Yeah a warning can be fine

## Follow up for nr.2
The way I use this is to override a hosted environment with some local values. So I combine this into a better developer experience to be able to adjust how one need to test the infrastructure. By default we run directly to dev env, but using .env default values we showcase how to override. But sometimes you dont want to make that file, since we cannot use the "template" file as secrets can be exposed. So we say, make a .env.local file with these values filled in to override. But again, sometimes one dont need that

**Comment by kaplanelad** [comment]
Hey @vongohren,
You can now set default values to `opts` section by adding comma after the env field
For example:
```yaml
opts:
  region: env:AWS_REGION, us-east-1
```
Thanks for the good idea

**Comment by vongohren** [comment]
@kaplanelad great! Did recongnize this with an emojie earlier, but now I have a question.

This is not released for 1.5.0 right?
Do you guys have any documentation on how to build and run the lastest version from github?
Or when is this release planned :)?

**Comment by kaplanelad** [comment]
Hey @vongohren, 
This feature just released now. you can update to version 1.5.1.
